packages/manager/src/queries/volumes/volumes.ts

```diff
@@ -121,6 +136,11 @@ export const useResizeVolumeMutation = () => {
   return useMutation<Volume, APIError[], ResizeVolumePayloadWithId>({
     mutationFn: ({ volumeId, ...data }) => resizeVolume(volumeId, data),
     onSuccess(volume) {
+      // Invalidate the specific volume
+      queryClient.invalidateQueries({
+        queryKey: volumeQueries.volume(volume.id).queryKey,
+      });
```

Comment:
Let's update the volume directly instead of doing an extra fetch

```suggestion
      // Update the specific volume
      queryClient.setQueryData<Volume>(volumeQueries.volume(volume.id).queryKey, volume),

```